packages/mui-joy/src/SvgIcon/SvgIcon.test.tsx

```diff
@@ -1,14 +1,19 @@
 import * as React from 'react';
 import { expect } from 'chai';
 import { createRenderer, describeConformance } from 'test/utils';
-import SvgIcon, { svgIconClasses as classes } from '@mui/joy/SvgIcon';
+import SvgIcon, {
+  svgIconClasses as classes,
+  SvgIconClassKey,
+  SvgIconProps,
+} from '@mui/joy/SvgIcon';
 import { ThemeProvider } from '@mui/joy/styles';
 import { unstable_capitalize as capitalize } from '@mui/utils';
 
 describe('<SvgIcon />', () => {
   const { render } = createRenderer();
 
-  let path;
+  type PathType = string | number | boolean | JSX.Element | React.ReactFragment | null | undefined;
+  let path: PathType;
```

Comment:
```suggestion
  let path: any;
```
this should be enough